app/models/comment.rb

```diff
@@ -0,0 +1,15 @@
+class Comment < ApplicationRecord
+  belongs_to :post, foreign_key: true, optional: true
+  belongs_to :user, foreign_key: true, optional: true
```

Comment:
- [ ] OPTIONAL 
(_However required for your code to work as intended_)

 Establishing the _Post_ **association** should be such that _Commentt_ belong to _User_ **through** _Author_, an alias of _User_, as per the given ERD, like below:
`belongs_to :author, class_name: 'User'`

  Kindly ensure to improve upon this aspect.